Content.Server/Materials/MaterialStorageSystem.cs

```diff
@@ -40,15 +40,20 @@ private void OnDeconstructed(EntityUid uid, MaterialStorageComponent component,
         }
     }
 
-    public override bool TryInsertMaterialEntity(EntityUid user, EntityUid toInsert, EntityUid receiver, MaterialStorageComponent? component = null)
+    public override bool TryInsertMaterialEntity(EntityUid user,
+        EntityUid toInsert,
+        EntityUid receiver,
+        MaterialStorageComponent? storage = null,
+        MaterialComponent? material = null,
+        PhysicalCompositionComponent? composition = null)
     {
-        if (!Resolve(receiver, ref component))
+        if (!Resolve(receiver, ref storage))
```

Comment:
could try to resolve all three components here, otherwise the base method will resolve these for you